package.json

```diff
@@ -22,12 +22,13 @@
   },
   "main": "node/index.js",
   "dependencies": {
+    "babel-core": "^5",
     "babel-runtime": "^5",
     "lodash": "^3.3.1",
+    "through": "^2.3.6",
     "tiny-sprintf": "^0.3.0"
   },
   "devDependencies": {
-    "babel-core": "^5",
```

Comment:
Oh, I'm blind. You added them to runtime dependencies.... mmh, not sure if I like that. There is no reason for "normal" users of JSNetworkX to have to install Babel. More information about what you were trying to do would be helpful.